libs/antsi/Cargo.toml

```diff
@@ -0,0 +1,7 @@
+[package]
```

Comment:
Wee need some further fields here:
- [authors](https://doc.rust-lang.org/cargo/reference/manifest.html#the-authors-fields) (`["HASH"]`? cc @nonparibus)
- [license](https://doc.rust-lang.org/cargo/reference/manifest.html#the-license-fields) (`"MIT OR Apache-2.0"` cc @hashintel/legal)
- [description](https://doc.rust-lang.org/cargo/reference/manifest.html#the-description-field)
- [readme](https://doc.rust-lang.org/cargo/reference/manifest.html#the-readme-field) (`"README.md"`)
- [repository](https://doc.rust-lang.org/cargo/reference/manifest.html#the-repository-field) (`"https://github.com/hashintel/hash/tree/main/libs/antsi"`)

Reply:
@TimDiekmann `authors = ["HASH"]`, please (mirroring `error-stack` - we might want to add this to `deer`, too)

Reply:
fixed in [e11f0f2](https://github.com/hashintel/hash/pull/2018/commits/e11f0f2311eba157f5813101fa749cd1e7d71ab1)

Reply:
You missed `authors = ["HASH"]` 🙂

Reply:
I would never! (fixed in [6682bb5](https://github.com/hashintel/hash/pull/2018/commits/6682bb5be3eeb0bb73922bafc68399fe5ded08fe)) :D